### PR TITLE
Transform called CommonJS imports

### PR DESCRIPTION
**Pull request by jsnmoon**
This adds support for transforming the following:

**Before**:
```javascript
var a = require( '...' )( ... );
``` 

**After**:
```javascript
import aFactory from '...'; 
const a = aFactory( ... );
```

### CODE REVIEW

File: transforms/cjs.js [hunk 1]
```diff
@@ -80,3 +95,47 @@ function replaceDeclarator (j, variableDeclarator) {
...
+		j(variableDeclarator.parent).remove();
+	} else {
+		if ('comments' in variableDeclarator.value) {
+			newImport.comments = variableDeclarator.value.comments;
+		}
+		j(variableDeclarator).remove();
+	}
+}
+
+function createImport(j, name, source) {
```
Comment (gmathieu) [inline transforms/cjs.js]: Question could you not use `util.createImportStatement()`?

Comment (jsnmoon) [inline transforms/cjs.js]: Yes, you're correct - let me correct this.

**Comment by jsnmoon** [comment]
@gmathieu: Would it be possible to do a 1.6.0 release including this change?

**Comment by jsnmoon** [comment]
Any chance of merging this PR?

**Comment by xjamundx** [comment]
Sorry for missing this. Yes we can get this in.

**Comment by xjamundx** [comment]
yeah looks good to me

**Pull request merged**